app/util/data_preparation/api/jira_clients.py

```diff
@@ -97,9 +98,15 @@ def issues_search(self, jql='order by key', start_at=0, max_results=1000, valida
         last_loop_remainder = max_results % BATCH_SIZE_ISSUES
 
         while loop_count > 0:
-            api_url = f'{self.host}/rest/api/2/search?jql={jql}&startAt={start_at}&maxResults={max_results}' \
-                      f'&validateQuery={validate_query}&fields={fields}'
-            response = self.get(api_url, "Could not retrieve issues")
+            api_url = f'{self.host}/rest/api/2/search'
```

Comment:
api_url variable is not changing in loop cycles, so it does not make sense to re-assign it on every loop. Let's move out it from a `while` statement.